rkyv_wrappers/src/tests.rs

```diff
@@ -1,29 +1,30 @@
-pub mod as_hashmap {
-    #[test]
-    fn struct_with_hashmap() {
-        use rkyv::{
-            archived_root,
-            ser::{serializers::AllocSerializer, Serializer},
-            Deserialize, Infallible,
-        };
+// pub mod as_hashmap {
```

Comment:
There's only tests for hashmap, so I commented them as I commented hashmap too ; let's focus on phantomdata first.